rust/nbabel-nbody/src/simd.rs

```diff
@@ -0,0 +1,127 @@
+//! Based on https://github.com/rust-lang/packed_simd/blob/master/examples/nbody/src/simd.rs
+use std::path::Path;
+use std::{fs, mem};
+
+use packed_simd::*;
```

Comment:
Can we use explicit import here? It seems that only `f64x4` is required.
```suggestion
use packed_simd::f64x4;
```

Reply:
You are right, I thought `f64x64` would need more traits imported from the root package.